### PR TITLE
TPC-QC: update of Tracks task

### PR DESCRIPTION
**Pull request by stheckel**
o Add histograms for q over pT
o Enlarge range of Ncluster histograms
o Change switch of A or C side from eta to Clusters only on that side

### CODE REVIEW

File: Detectors/TPC/qc/src/Tracks.cxx [hunk 1]
```diff
@@ -50,17 +50,21 @@ void Tracks::initializeHistograms()
   mHist1D.emplace_back("hPtPos", "Transverse momentum, pos. tracks;p_T", logPtBinning.size() - 1, logPtBinning.data());         //| mHist1D[14]
   mHist1D.emplace_back("hEtaBeforeCuts", "Pseudorapidity (before cuts);eta", 400, -2., 2.);                                     //| mHist1D[15]
   mHist1D.emplace_back("hPtBeforeCuts", "Transverse momentum (before cuts);p_T", logPtBinning.size() - 1, logPtBinning.data()); //| mHist1D[16]
+  mHist1D.emplace_back("hQOverPt", "Charge over transverse momentum;q/p_T", 400, -20., 20.);                                    //| mHist1D[17]
+  mHist1D.emplace_back("hPhiBothSides", "Azimuthal angle, both sides clusters;phi", 360, 0., 2 * M_PI);                         //| mHist1D[18]
```
Comment (chiarazampolli) [inline Detectors/TPC/qc/src/Tracks.cxx:54]: Is this used? Maybe I just don't see it in the changes?

Comment (wiechula) [reply Detectors/TPC/qc/src/Tracks.cxx:54]: It is in lines 127 and 132 below.

**Review by wiechula — DISMISSED** [review]

File: Detectors/TPC/qc/src/Tracks.cxx (hunk 1 above)
Comment (chiarazampolli) [reply Detectors/TPC/qc/src/Tracks.cxx:54]: Ah yes, I did not see L127. Thanks!
BTW, we need @shahor02 or @davidrohr to approve...

File: Detectors/TPC/qc/src/Tracks.cxx [hunk 2]
```diff
@@ -103,44 +111,53 @@ bool Tracks::processTrack(const o2::tpc::TrackTPC& track)
   // ===| histogram filling including cuts |===
   if (absEta < 1. && nCls > 60 && dEdxTot > 20) {
 
+    if (pt > 0) {
```
Comment (shahor02) [inline Detectors/TPC/qc/src/Tracks.cxx]: Why don't you use directly `track.getQ2Pt()` ?

**Review by shahor02 — COMMENTED** [review]
Hi, a small comment below, but since the changes are rather trivial, I can merge it w/o waiting for remaining CIs, let me know.

**Comment by stheckel** [comment]
Thanks Chiara and Jens for checking this PR!

Concerning the usage of the histograms, honestly, the implementation with the vectors is a bit ugly at the moment and with more and more histograms its indeed getting difficult to keep track. We are already working on restructuring the PID task using maps instead of vectors, this could/should be also done for the Tracks (but not in this PR).

File: Detectors/TPC/qc/src/Tracks.cxx (hunk 2 above)
Comment (stheckel) [reply Detectors/TPC/qc/src/Tracks.cxx]: Good point, I will change that.

**Comment by stheckel** [comment]
Hi @shahor02,
I updated this PR with your suggestion, which in principle is a trivial change. Before, almost all checks had been done and successful, now I see that the checks take much longer, most are running or not even started and one even failed - with the error "No errors found", so I guess unrelated. Would it maybe still be possible to merge?

**Review by shahor02 — APPROVED** [review]

**Comment by stheckel** [comment]
Thanks a lot!

**Pull request merged**